festim/hydrogen_transport_problem.py

```diff
@@ -368,48 +385,52 @@ def assign_functions_to_species(self):
     def define_markers_and_measures(self):
         """Defines the markers and measures of the model"""
```

Comment:
Maybe we can add more details to these docstrings like the fact that we set the facet_meshtags and volume_meshtags attributes as well as dx and ds